src/main/java/com/internet/shop/controllers/order/DeleteOrderController.java

```diff
@@ -0,0 +1,27 @@
+package com.internet.shop.controllers.order;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.service.OrderService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/orders/delete")
+public class DeleteOrderController extends HttpServlet {
+    private static final Long ORDER_ID = 1L;
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private OrderService orderService =
+            (OrderService) injector.getInstance(OrderService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String id = req.getParameter("id");
+        Long userId = Long.valueOf(id);
+        orderService.delete(ORDER_ID);
```

Comment:
can you explain please why are you receiving userId as request parameter
where do you use it and why is orderId hardcoded?